Firmware/printer.cfg

```diff
@@ -83,108 +112,120 @@ uart_pin: PC11
 tx_pin: PC10
 uart_address: 1
 interpolate: True
-#run_current: 0.37       # For V0.1 spec NEMA17 LDO-42STH25-1004CL200E 1.0A  
+## For OMC (StepperOnline) 17LS13-0404E-200G 0.4A 
+#run_current: 0.2
+#hold_current: 0.15
+## For LDO-42STH25-1004CL200E 1.0A
+#run_current: 0.37
 #hold_current: 0.35
-run_current: 0.2         # For V0.1 spec NEMA17 OMC 17LS13-0404E-200G 0.4A 
```

Comment:
@newphreak You now have NO `run_current` specified at all for the Z stepper!

Reply:
This was intentional to make the user choose the correct one and not trust anything blindly

Reply:
it is also stated clearly in the top what you should look at. ideas to make it clearer are more then welcome